src/commands/misc/chocolatey.js

```diff
@@ -0,0 +1,35 @@
+const { SearchCommand, SwitchbladeEmbed, Constants } = require('../..')
+
+module.exports = class Chocolatey extends SearchCommand {
+  constructor (client) {
+    super({
+      name: 'chocolatey',
+      aliases: ['choco'],
+      requirements: { apis: ['chocolatey'] },
+      embedColor: Constants.CHOCOLATEY_COLOR,
+      embedLogoURL: 'https://i.imgur.com/CRddVay.png'
+    }, client)
+  }
+
+  async search (_, query) {
+    const res = await this.client.apis.chocolatey.search(query)
+    // TODO: figure out why sometimes the api returns JSON and other times it returns XML (eg.: vim and git, respectively)
+    return res.data.d.results
+  }
+
+  searchResultFormatter (i) {
+    return `[${i.Title} - ${i.Authors}](${i.GalleryDetailsUrl})`
+  }
+
+  async handleResult ({ t, author, channel, language }, { GalleryDetailsUrl, IconUrl, Title, Summary }) {
+    channel.send(
+      new SwitchbladeEmbed(author)
+        .setColor(this.embedColor)
+        .setAuthor(t('commmands:chocolatey.chocolatey'), this.embedLogoURL, 'https://chocolatey.org/packages')
```

Comment:
you typed commands with three "m"

Reply:
hmmm

Reply:
fixed in https://github.com/SwitchbladeBot/switchblade/pull/1166/commits/3529edfd90dce302ae077dbf033a413ed4aaaae7